framework/test/unit/modules/reward/api.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { getRandomBytes, hash } from '@liskhq/lisk-cryptography';
+import { RewardModule } from '../../../../src/modules/reward';
+import { EventQueue } from '../../../../src/node/state_machine';
+
+describe('RewardModuleAPI', () => {
+	const genesisConfig: any = {};
+	const moduleConfig: any = {
+		distance: 3000000,
+		offset: 2160,
+		brackets: [
+			BigInt('500000000'), // Initial Reward
+			BigInt('400000000'), // Milestone 1
+			BigInt('300000000'), // Milestone 2
+			BigInt('200000000'), // Milestone 3
+			BigInt('100000000'), // Milestone 4
+		],
+		tokenIDReward: { chainID: 0, localID: 0 },
+	};
+	const generatorConfig: any = {};
+
+	const context = {
```

Comment:
There is a bug in `framework/src/testing/create_contexts.ts` but it's good to rename `createAPIContext` to `create TransientAPIContext`, and use that function. (Other test will require this function as well)

Reply:
solved and done